docs/admin/cluster-management.md

```diff
@@ -136,7 +136,7 @@ The initial values of the autoscaler parameters set by ``kube-up.sh`` and some m
 or using gcloud CLI:
 
 ```
-gcloud preview autoscaler --zone compute-zone <command>
+gcloud alpha compute autoscaler --zone $ZONE <command>
```

Comment:
Also, here $ZONE assumes that the environment variable is set (is that correct?) but in this topic and after searching against the repo, we dont specify how to set that variable.